lib/buchungsstreber/cli/app.rb

```diff
@@ -208,6 +208,7 @@ def watch(date = nil)
       method_option :date, :default => 'today'
       def add(*entry)
         date = parse_date(options[:date])
+        entry << "# Added on " + Time.at(Time.now.to_i).to_s
```

Comment:
I think it would be less overhead to only show a limited time format.

```ruby
Time.now.strftime("%Y-%m-%d %H:%M")
```

perhaps.


Also, I think it would be better to use `"(added on #{time})"` since the `#` is already used as a comment sign elsewhere.